README.md

```diff
@@ -182,7 +182,7 @@ render((
 ```
 
 #### Form error event handler
-
+z
```

Comment:
```suggestion

```

Reply:
I think I was thinking about sleeping? LOL - thanks for catching this 😝